stories/CCard.stories.ts

```diff
@@ -184,3 +205,43 @@ export const Color: Story = {
     slotTemplate: "Cards main content.",
   }),
 };
+
+export const ScrollableContent: Story = {
+  args: {
+    title: "Scrollable Card content",
+    subtitle:
+      "Scrolling will be automatically enabled if the content is larger than the available space. This only works on a CCard with fixed or a maximum height. Keep in mind, that the surrounding divs also have to have a fixed or maximum height and also have the height set to 100%.",
+    height: "100%",
+  },
+  render: createStorybookRender({
+    components: { CCard },
+    template: `
+<div style="height: 100%; max-height: 300px;" class="pa-5">
+  <c-card v-bind='args'>
+    <template #content-full-width="{ padding }">
+        <v-list>
+            <v-list-item :class="padding" @click="() => {}">This content has the full width</v-list-item>
+            <v-divider></v-divider>
+            <v-list-item :class="padding" @click="() => {}">This content has the full width</v-list-item>
+            <v-divider></v-divider>
+            <v-list-item :class="padding" @click="() => {}">Use 'padding' prop to set matching padding.</v-list-item>
+        </v-list>
+    </template>
+  </c-card>
+</div>
+    `,
+  }),
+  parameters: createStorybookParameters({
+    slotTemplate: `
+      <template #content-full-width="{ padding }">
+          <v-list>
+              <v-list-item :class="padding">This content has the full width</v-list-item>
+              <v-divider></v-divider>
+              <v-list-item :class="padding" @click="() => {}">This content has the full width</v-list-item>
+              <v-divider></v-divider>
+              <v-list-item :class="padding">Use 'padding' prop to set matching padding.</v-list-item>
```

Comment:
its still not consisten with the code block above

Reply:
yea i know, that's because I didn't want to fill the code with unnessecary code which is not required for functionality and does not affect the overall look/layout.

The @click was only added to get the "hover" effect. Do you think, we really need to add this to the code block?

Reply:
IMO the code block should be exactly what you need to do to get the same result. Yes.

Reply:
done